src/apis/btc-relay.ts

```diff
@@ -10,6 +11,7 @@ export interface BTCRelayAPI {
     getLatestBlock(): Promise<H256Le>;
     getLatestBlockHeight(): Promise<u32>;
     verifyTransactionInclusion(txid: string, confirmations?: number, insecure?: boolean): Promise<void>;
+    getParachainBlockHeight(): Promise<BlockNumber>;
```

Comment:
We have this query already in the `system` API: https://github.com/interlay/polkabtc-js/blob/dev/src/apis/system.ts#L11